client/flutter/test/travel_advice_test.dart

```diff
@@ -0,0 +1,95 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_localizations/flutter_localizations.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:WHOFlutter/generated/l10n.dart';
+import 'package:WHOFlutter/pages/travel_advice.dart';
+
+enum Direction { LEFT, RIGHT }
+
+// Reusable swiping function.
+void swipe(WidgetTester tester, [Direction direction = Direction.LEFT]) async {
```

Comment:
This only works for PageViews so it might be worth renaming the function or allowing the type of scrollable to be provided.

Reply:
Makes sense. I've renamed it to `swipePageView`